### PR TITLE
Hungarian language update #14

### PR DESCRIPTION
**Pull request by big4billy**
* Last fuzzy removals
* New translations additions

### CODE REVIEW

**Comment by big4billy** [comment]
> Hi, @big4billy. I left some comments. Please have a look when you have time.
> 
> Note that you have done a lot of changes to English strings. Since your changes reverts the English strings to older variants, I suppose this is because you were on an outdated PO file and then probably resolved merge conflicts and kept the old variants.

I use this .PO since September 2023 and I usually do click on syncronize (as I did before this pull request). It supposed to contain the newest English titles, isn't it?
I tested all the changes with the current 1.0.11 version and all the translations got shown right. If this methode is wrong please explain me detailed how I could earn these texts to be shown translated? What I should revert these texts for?

File: files/lang/hu.po [hunk 1]
```diff
@@ -325,21 +324,21 @@ msgid ""
...
-"Több száz\n"
+"Többszáz\n"
 "%{monster}"
 
 msgid ""
 "Zounds...\n"
 "%{monster}"
 msgstr ""
-"Ezrednyi\n"
+"Ezernyi\n"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Changed to "Seregnyi"

File: files/lang/hu.po [hunk 2]
```diff
@@ -368,7 +367,7 @@ msgid "army|Throng"
 msgstr "Tömeg"
 
 msgid "army|Swarm"
-msgstr "Század"
+msgstr "Többszáz"
 
 msgid "army|Zounds"
 msgstr "Ezred"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po:373]: "Többszáz" is OK, just like in the Hungarian translation of HoMM IV.

File: files/lang/hu.po [hunk 3]
```diff
@@ -7486,12 +7558,11 @@ msgid ""
...
 msgstr ""
 "%{skill} segítségével a csatában megölt lények %{count} százalékából "
-"Csontvázakat igézhetünk meg."
+"Csontvázakat idézhetünk meg."
 
-#, fuzzy
 msgid ""
-"The hero produces %{count} gold pieces per day as tax revenue from estates."
-msgstr "A hősöd napi %{count} arany adóbevételt kap."
+"Your hero produces %{count} gold pieces per day as tax revenue from estates."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 4]
```diff
@@ -7352,132 +7442,114 @@ msgstr "Szakértő vagyon"
...
 msgstr ""
-"A(z) %{skill} %{count} százaléknyi esélyt ad a hősnek, hogy az ellenfél "
+"%{skill} %{count} százaléknyi esélyt ad a hősnek, hogy az ellenfél "
 "által a csatában használt 3-as (vagy alacsonyabb) szintű varázslatot "
 "tanuljon el."
 
-#, fuzzy
 msgid ""
-"%{skill} gives the hero a %{count} percent chance to learn any given 4th "
+"%{skill} gives your hero a %{count} percent chance to learn any given 4th "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 5]
```diff
@@ -7352,132 +7442,114 @@ msgstr "Szakértő vagyon"
...
-#, fuzzy
-msgid "All of the creatures may offer to join the hero."
-msgstr "Az összes lény csatlakozhat hozzád."
+msgid "All of the creatures may offer to join you."
+msgstr "Az összes lény csatlakozhat a hős seregéhez."
 
-#, fuzzy
 msgid ""
-"%{skill} increases the hero's movement points over water by %{count} percent."
+"%{skill} increases your hero's movement points over water by %{count} percent."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 6]
```diff
@@ -7352,132 +7442,114 @@ msgstr "Szakértő vagyon"
...
 msgstr ""
-"A(z) %{skill} lehetővé teszi a seregednél gyengébb lényekkel való "
+"A(z) %{skill} lehetővé teszi a hős seregénél gyengébb lényekkel való "
 "tárgyalást. "
 
-#, fuzzy
 msgid ""
-"Approximately %{count} percent of the creatures may offer to join the hero."
-msgstr "A lények kb. %{count} százaléka csatlakozhat hozzád."
+"Approximately %{count} percent of the creatures may offer to join you."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 7]
```diff
@@ -7352,132 +7442,114 @@ msgstr "Szakértő vagyon"
...
 "by %{count} percent."
 msgstr ""
-"A(z) %{skill} növeli a távolsági támadások sebzését %{count} százalékkal."
+"%{skill} növeli a hős távolsági támadással rendelkező lényeinek "
+"sebzését %{count} százalékkal."
 
-#, fuzzy
-msgid "%{skill} increases the hero's movement points by %{count} percent."
-msgstr "A(z) %{skill} növeli a hősöd mozgáspontjait %{count} százalékkal."
+msgid "%{skill} increases your hero's movement points by %{count} percent."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 8]
```diff
@@ -7352,132 +7442,114 @@ msgstr "Szakértő vagyon"
...
 
 msgid "%{skill} eliminates the movement penalty for rough terrain."
 msgstr ""
-"A(z) %{skill} kiiktatja a nehéz terepen történő haladásból fakadó hátrányt."
+"%{skill} kiiktatja a nehéz terepen történő haladásból fakadó hátrányt."
 
-#, fuzzy
 msgid ""
-"%{skill} increases the damage done by the hero's range attacking creatures "
+"%{skill} increases the damage done by range attacking creatures "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 9]
```diff
@@ -2520,7 +2517,7 @@ msgstr ""
 "árokba, ott be kell fejezze a körét és sebezhetőbbé válik."
 
 msgid ""
-"The Castle improves the town's defense and increases its income to %{count} "
+"The Castle improves town defense and increases income to %{count} "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 10]
```diff
@@ -7352,132 +7442,114 @@ msgstr "Szakértő vagyon"
...
+msgid "%{skill} increases your hero's movement points by %{count} percent."
+msgstr "%{skill} növeli a hősöd mozgáspontjait %{count} százalékkal."
 
-#, fuzzy
-msgid "%{skill} increases the hero's viewable area by one square."
-msgid_plural "%{skill} increases the hero's viewable area by %{count} squares."
-msgstr[0] "A(z) %{skill} növeli a hős láthatárát egy egységgel."
-msgstr[1] "A(z) %{skill} növeli a hős láthatárát %{count} egységgel."
+msgid "%{skill} increases your hero's viewable area by one square."
+msgid_plural "%{skill} increases your hero's viewable area by %{count} squares."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 11]
```diff
@@ -7352,132 +7442,114 @@ msgstr "Szakértő vagyon"
...
+msgstr[0] "%{skill} növeli a hős látóhatárát egy egységgel."
+msgstr[1] "%{skill} növeli a hős látóhatárát %{count} egységgel."
 
-#, fuzzy
 msgid ""
-"%{skill} allows the hero to negotiate with monsters who are weaker than "
-"their army."
+"%{skill} allows you to negotiate with monsters who are weaker than "
+"your group. "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 12]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
-#, fuzzy
 msgid ""
 "Approximately %{count} percent of the creatures may offer to join the hero."
-msgstr "A lények kb. %{count} százaléka csatlakozhat hozzád."
+msgstr "A lények kb. %{count} százaléka csatlakozhat a hős seregéhez."
 
-#, fuzzy
-msgid "All of the creatures may offer to join the hero."
-msgstr "Az összes lény csatlakozhat hozzád."
+msgid "All of the creatures may offer to join you."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 13]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 msgid ""
 "%{skill} increases the hero's movement points over water by %{count} percent."
 msgstr ""
-"A(z) %{skill} növeli a hős vízen történő mozgáspontjait %{count} százalékkal."
+"%{skill} növeli a hős vízen történő mozgáspontjait %{count} százalékkal."
 
-#, fuzzy
-msgid "%{skill} increases the hero's troops morale by %{count}."
-msgstr "A(z) %{skill} ennyivel növeli a csapatod tagjainak morálját: %{count}."
+msgid "%{skill} increases your hero's troops morale by %{count}."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 14]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
+msgstr "%{skill} ennyivel növeli a hős seregének morálját: %{count}."
 
-#, fuzzy
-msgid "%{skill} allows the hero to learn third level spells."
+msgid "%{skill} allows your hero to learn third level spells."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 15]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 msgstr ""
-"A(z) %{skill} képessé teszi a hősödet 3. szintű varázslatok megtanulására."
+"%{skill} képessé teszi a hősödet 3. szintű varázslatok megtanulására."
 
-#, fuzzy
-msgid "%{skill} allows the hero to learn fourth level spells."
+msgid "%{skill} allows your hero to learn fourth level spells."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 16]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 msgstr ""
-"A(z) %{skill} képessé teszi a hősödet 4. szintű varázslatok használatára."
+"%{skill} képessé teszi a hősödet 4. szintű varázslatok megtanulására."
 
-#, fuzzy
-msgid "%{skill} allows the hero to learn fifth level spells."
+msgid "%{skill} allows your hero to learn fifth level spells."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 17]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 msgstr ""
-"A(z) %{skill} képessé teszi a hősödet 5. szintű varázslatok használatára."
+"%{skill} képessé teszi a hősödet 5. szintű varázslatok megtanulására."
 
-#, fuzzy
-msgid "%{skill} regenerates one additional spell point per day to the hero."
+msgid "%{skill} regenerates one additional spell point per day to your hero."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 18]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 msgid_plural ""
-"%{skill} regenerates %{count} additional spell points per day to the hero."
+"%{skill} regenerates %{count} additional spell points per day to your hero."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 19]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
-"A(z) %{skill} naponta eggyel növeli a varázspontok újratermelődését."
+"%{skill} naponta eggyel növeli a hős varázspontjainak újratermelődését."
 msgstr[1] ""
-"A(z) %{skill} naponta %{count}-vel növeli a varázspontok újratermelődését."
+"%{skill} naponta %{count}-vel növeli a hős varázspontjainak újratermelődését."
 
-#, fuzzy
-msgid "%{skill} increases the hero's luck by %{count}."
-msgstr "A(z) %{skill} ennyivel növeli a hős szerencséjét: %{count}."
+msgid "%{skill} increases your hero's luck by %{count}."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 20]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
+msgstr "%{skill} ennyivel növeli a hős szerencséjét: %{count}."
 
-#, fuzzy
 msgid ""
-"%{skill} gives the hero's catapult shots a greater chance to hit and do "
+"%{skill} gives your hero's catapult shots a greater chance to hit and do "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 21]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 "damage to castle walls."
 msgstr ""
-"A(z) %{skill} nagyobb esélyt ad a hősöd katapultjának a találatra és a "
+"%{skill} nagyobb esélyt ad a hősöd katapultjának a találatra és a "
 "várfal rombolására."
 
-#, fuzzy
 msgid ""
-"%{skill} gives the hero's catapult an extra shot, and each shot has a "
+"%{skill} gives your hero's catapult an extra shot, and each shot has a "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 22]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 "greater chance to hit and do damage to castle walls."
 msgstr ""
-"A(z) %{skill} egy plusz lövést és nagyobb esélyt ad a hősöd katapultjának a "
+"%{skill} egy plusz lövést és nagyobb esélyt ad a hősöd katapultjának a "
 "találatra és a várfal rombolására."
 
-#, fuzzy
 msgid ""
-"%{skill} gives the hero's catapult an extra shot, and each shot "
+"%{skill} gives your hero's catapult an extra shot, and each shot "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 23]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 "automatically destroys any wall, except a fortified wall in a Knight castle."
 msgstr ""
-"A(z) %{skill} egy plusz lövést ad a hősöd katapultjának. Minden lövés "
+"%{skill} egy plusz lövést ad a hősöd katapultjának. Minden lövés "
 "automatikusan rombolja a falat, kivéve a Lovagvár sánccal megerősített várát."
 
-#, fuzzy
 msgid ""
-"%{skill} gives the hero a %{count} percent chance to learn any given 1st or "
+"%{skill} gives your hero a %{count} percent chance to learn any given 1st or "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

File: files/lang/hu.po [hunk 24]
```diff
@@ -7409,132 +7497,114 @@ msgstr "Szakértő vagyon"
...
 "2nd level spell that was cast by an enemy during combat."
 msgstr ""
-"A(z) %{skill} %{count} százaléknyi esélyt ad a hősnek, hogy az ellenfél "
+"%{skill} %{count} százaléknyi esélyt ad a hősnek, hogy az ellenfél "
 "által a csatában használt 1-es vagy 2-es szintű varázslatot tanuljon el."
 
-#, fuzzy
 msgid ""
-"%{skill} gives the hero a %{count} percent chance to learn any given 3rd "
+"%{skill} gives your hero a %{count} percent chance to learn any given 3rd "
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Reverted

**Comment by ihhub** [comment]
Hi @big4billy , can you please fix merge conflicts so I can proceed with the review?

**Comment by big4billy** [comment]
> Hi @big4billy , can you please fix merge conflicts so I can proceed with the review?

Solved, with some fuzzy removals in the same topic nearby.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@big4billy , thank you so much for these changes!

**Pull request merged**